src/service/url-expander/expander.js

```diff
@@ -140,11 +140,13 @@ export class Expander {
               // precedence of resolution choices in #expandBinding_ later.
               name: match.name,
               prioritized: this.bindings_[match.name],
+              noEncode,
             };
           } else {
             // or the global source
             binding = this.variableSource_.get(match.name);
             binding.name = match.name;
+            binding.noEncode = noEncode;
```

Comment:
wait, is this going to change the binding in `this.variableSource_`? or is it a copy?